x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/display_settings/display_settings_logic.ts

```diff
@@ -24,6 +24,7 @@ const SUCCESS_MESSAGE = 'Display Settings have been successfuly updated.';
 
 import { DetailField, SearchResultConfig, OptionValue, Result } from '../../../../types';
 
+export const UNASSIGNED_FIELD = 'Leave unassigned';
```

Comment:
Can you go ahead and create a `constants.ts` file in the this folder and move this here? 

```JavaScript
x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/display_settings/constants.ts
```
Also, let's go ahead and add i18n to this constant. You can view the patterns [here](https://github.com/elastic/kibana/blob/1f310f780d098b1229629df507439aae4c8e5522/x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/constants.ts).